[2024-10-06T17:23:58.309+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-06T17:23:59.290+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.choose_source manual__2024-10-07T00:23:33+07:00 [queued]>
[2024-10-06T17:23:59.316+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.choose_source manual__2024-10-07T00:23:33+07:00 [queued]>
[2024-10-06T17:23:59.322+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-06T17:23:59.379+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): choose_source> on 2024-10-06 17:23:33+00:00
[2024-10-06T17:23:59.420+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=253) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-06T17:23:59.425+0000] {standard_task_runner.py:72} INFO - Started process 255 to run task
[2024-10-06T17:23:59.426+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'choose_source', 'manual__2024-10-07T00:23:33+07:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp3n9fgklf']
[2024-10-06T17:23:59.446+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask choose_source
[2024-10-06T17:23:59.677+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.choose_source manual__2024-10-07T00:23:33+07:00 [running]> on host 0cbddbe17999
[2024-10-06T17:24:00.133+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='choose_source' AIRFLOW_CTX_EXECUTION_DATE='2024-10-06T17:23:33+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-07T00:23:33+07:00'
[2024-10-06T17:24:00.158+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-06T17:24:00.372+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-06T17:24:00.385+0000] {logging_mixin.py:190} INFO - Selected source type: csv
[2024-10-06T17:24:00.386+0000] {python.py:240} INFO - Done. Returned value was: extract_csv
[2024-10-06T17:24:00.387+0000] {branch.py:38} INFO - Branch into extract_csv
[2024-10-06T17:24:00.388+0000] {skipmixin.py:230} INFO - Following branch ('extract_csv',)
[2024-10-06T17:24:00.462+0000] {skipmixin.py:278} INFO - Skipping tasks [('extract_json', -1)]
[2024-10-06T17:24:00.662+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-06T17:24:00.664+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=choose_source, run_id=manual__2024-10-07T00:23:33+07:00, execution_date=20241006T172333, start_date=20241006T172359, end_date=20241006T172400
[2024-10-06T17:24:00.867+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-06T17:24:01.090+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-06T17:24:01.105+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
